x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -108,6 +108,7 @@ export enum SecurityPageName {
   overview = 'overview',
   policies = 'policy',
   rules = 'rules',
+  rulesCreate = 'rules_create',
```

Comment:
NIT: maybe the convention would be `rules-create`

Reply:
Inside `SecurityPageName`, there are `17` usages of `_` and 12 of `-`. So I don't know which one is the convention here. 🤷 

> Conventions are great because everyone has a different one.

Reply:
`-` is used to prefix the main section in sub-sections names (see `network-dns`, `hosts-events`), `_` is used to separate words (`event_filters`, `network-external_alerts`)

Reply:
Ok, It makes some sense. I will follow this rule.